spec/asyncapi.md

```diff
@@ -1439,11 +1439,15 @@ Field Name | Type | Description
 ---|:---|--- 
 <a name="componentsSchemas"></a> schemas | Map[`string`, [Schema Object](#schemaObject) \| [Reference Object](#referenceObject)] | An object to hold reusable [Schema Objects](#schemaObject).
 <a name="componentsServers"></a> servers | Map[`string`, [Server Object](#serverObject) \| [Reference Object](#referenceObject)] | An object to hold reusable [Server Objects](#serverObject).
-<a name="componentsChannels"></a> channels | Map[`string`, [Channel Item Object](#channelItemObject)] | An object to hold reusable [Channel Item Objects](#channelItemObject).
+<a name="componentsChannels"></a> channels | Map[`string`, [Channel Item Object](#channelItemObject)] \| [Reference Object](#referenceObject)] | An object to hold reusable [Channel Item Objects](#channelItemObject).
+<a name="componentsOperations"></a> operations | Map[`string`, [Operation Item Object](#operationObject)] \| [Reference Object](#referenceObject)] | An object to hold reusable [Operation Item Objects](#operationObject).
```

Comment:
@magicmatatjahu would you mind removing `operations` from this PR? I already moved them in this one: https://github.com/asyncapi/spec/pull/806.

Reply:
@fmvilas Done :)